### ISSUE TITLE
Publish the new release: 1.0.0-alpha1

### ISSUE THREAD

**Issue by Hywan** [wasmerio/wasmer-python#262]
I'm writing this issue to provide a summary of where we are at publishing the 1.0.0-alpha1 version.

The rewrite is over. We are going to publish the new release. To do so, we need to fix an issue with LLVM.

LLVM is required for the `wasmer_compiler_llvm` crate. So far, we use the official LLVM builts on Linux and macOS. On Windows, we are using a custom built from ours (see https://github.com/wasmerio/llvm-build/).

However, the official LLVM builts come with ncurse as a dependency. It breaks the `manylinux` rule of Python wheels. Our own builts have ncurse turned off, so it is likely to fix that issue! However, we have another issue: Our own builts do not build `clang`, which is required by the native engine.

This last problem is being tested in https://github.com/wasmerio/llvm-build/pull/1 for the moment.

Once it is done, we will see if our custom builts will fix the issue with Python wheels.

**Comment by syrusakbary** [comment]
The clang thing failed after merging https://github.com/wasmerio/llvm-build/pull/1

https://dev.azure.com/wasmerio/llvm-build/_build/results?buildId=4833&view=logs&j=51a345e8-f255-5928-6ab3-8fe0ea15a0d4&t=d7608f98-0737-5fe4-3bc5-2553ced25990
:(

**Comment by Hywan** [comment]
I've developed, with the help of @jubianchi, a new build workflow, https://github.com/Hywan/llvm-custom-builds. It works, and we already have the LLVM 10.x build packages. Let's try it out.
